### PR TITLE
User/amagraw/fix milli bytes bug

### PR DESCRIPTION
**Pull request by pfrcks**

### CODE REVIEW

**Comment by pfrcks** [comment]
image

**Review by ganga1980 — APPROVED** [review]

File: source/plugins/ruby/KubernetesApiClient.rb [hunk 1]
```diff
@@ -751,8 +751,13 @@ def getMetricNumericValue(metricName, metricVal)
             metricValue.chomp!("k")
             metricValue = Float(metricValue) * 1000.0 ** 1
           elsif (metricValue.end_with?("m"))
+            #original value before downcase ending with M is megabyte and value ending with m is milli-byte
```
Comment (vishiy) [inline source/plugins/ruby/KubernetesApiClient.rb:754]: Amol, change looks ok. But please log a future work-item to make this function all case-insensitive, so we can add support for future units easily.

**Review by vishiy — APPROVED** [review]

File: source/plugins/ruby/KubernetesApiClient.rb [hunk 2]
```diff
@@ -751,8 +751,13 @@ def getMetricNumericValue(metricName, metricVal)
...
             metricValue.chomp!("m")
-            metricValue = Float(metricValue) * 1000.0 ** 2
```
Comment (vishiy) [inline source/plugins/ruby/KubernetesApiClient.rb:755]: also, can you check if Ux (live) supportes this unit ? if not we need to fix that as well.

Comment (pfrcks) [reply source/plugins/ruby/KubernetesApiClient.rb:755]: sure

**Pull request merged**